web/client/components/widgets/builder/wizard/map/enhancers/__tests__/withSelectedNode-test.js

```diff
@@ -5,10 +5,10 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import React from'react';
```

Comment:
if you want to avoid this lint problem i can suggest you to (if you are using vs code) to enabled the ESLint extension
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint